storage/storage.go

```diff
@@ -1,9 +1,19 @@
 package storage
 
-import "io"
+import (
+	"io"
+	"os"
+)
+
+type File struct {
```

Comment:
I'll update from `File` to `FileEntry` and we can see what @bradrydzewski 's input is to this all in general.  :)